js/components/home.tsx

```diff
@@ -1,24 +1,73 @@
 import { OperatorSignInModal } from "./operatorSignIn/operatorSignInModal";
-import { ReactElement } from "react";
+import { DateTime } from "luxon";
+import { ReactElement, useMemo, useState } from "react";
 import { Link } from "react-router-dom";
 
 export const Home = (): ReactElement => {
+  const [modalOpen, setModalOpen] = useState(false);
+  // check once to avoid it changing while looking at the page
+  // no attempt yet to verify time zones / rolling the date at a time other than midnight
+  const today = useMemo<string>(() => DateTime.now().toISODate(), []);
+  const [selectedDate, setSelectedDate] = useState<string>(today);
   return (
-    <>
-      <div className="text-3xl">
-        🪐
-        <span className="text-mbta-orange">O</span>
-        <span className="text-mbta-red">r</span>
-        <span className="text-mbta-blue">b</span>it
-        <div>
-          <Link to="/list">
-            <button className="bg-mbta-blue text-gray-100 rounded-md p-2 text-sm m-5">
-              Sign-in history
-            </button>
-          </Link>
-        </div>
+    <div className="max-w-lg mx-auto">
+      <div className="flex justify-between">
+        <hgroup className="mb-3">
+          <h1 className="text-xl">Operators</h1>
+          <p>Search and sign in operators</p>
+        </hgroup>
+        <button
+          className="rounded bg-[#1c1e23] hover:bg-[#404040] text-white p-1 self-center"
+          onClick={() => {
+            setModalOpen(true);
+          }}
+        >
+          Sign In Operator
+        </button>
       </div>
-      <OperatorSignInModal />
-    </>
+      <div className="flex justify-between gap-2 items-end mb-3">
+        <label className="flex flex-col">
+          <span className="text-sm">Service Date</span>
+          <input
+            type="date"
+            value={selectedDate}
+            onChange={(evt) => {
+              console.log(evt.target.value);
+              setSelectedDate(evt.target.value);
+            }}
+            className="rounded"
+          />
+        </label>
+        <button
+          className="rounded border border-[#929292] hover:bg-[#F6F6F6] px-2 py-1 disabled:text-[#929292]"
```

Comment:
**nitpick:** #929292 at least exists in out color palette defined in the Tailwind config and doesn't need to be directly given as a hex value here. I don't see #f6f6f6 but we should probably add that as well.

Reply:
Same goes for a handful of other hex values throughout the file.

Reply:
Looks like this has been addressed, thanks!